index.yaml

```diff
@@ -1,6 +1,29 @@
 apiVersion: v1
 entries:
   agent:
+  - apiVersion: v2
+    appVersion: 3.1.0
+    created: "2023-11-14T20:28:59.863924964+06:00"
+    description: Helm chart for the Optimizely Agent https://docs.developers.optimizely.com/full-stack/docs/optimizely-agent
+    digest: 8c95f9dad10753db0d94d56fa25d65c6514e2c07f4df133638f8c799c5260d6f
+    home: https://docs.developers.optimizely.com/full-stack/docs/optimizely-agent
+    icon: https://github.com/optimizely/optimizely-agent-helm/Optimzely-icon.png
+    keywords:
+    - Optimizely
+    - Optimizely Agent
+    - Full Stack
+    - Rollouts
+    - Feature Flags
+    maintainers:
+    - email: devx@optimizely.com
+      name: Optimizely Dev X Team
+    name: agent
+    sources:
+    - https://github.com/optimizely/agent
+    type: application
+    urls:
+    - https://optimizely.github.io/optimizely-agent-helm/packages/agent-1.2.0.tgz
```

Comment:
I get a 404 for this tarball. Does it work for you?

Reply:
This will be a valid URL after gh-page is successfully deployed.